pkg/make_rpm.py

```diff
@@ -225,17 +209,18 @@ def SetupWorkdir(self, spec_file, original_dir):
   def CallRpmBuild(self, dirname):
     """Call rpmbuild with the correct arguments."""
 
+    buildroot = os.path.join(dirname, RpmBuilder.BUILD_DIR)
```

Comment:
This is the only interesting part of the change. It changes --builtroot from <tmpdir>/BUILDROOT to <tmpdir>/BUILD. Without that, it was not finding any of my sources.